init-tools.cmd

```diff
@@ -46,15 +46,17 @@ if exist "%DotNetBuildToolsDir%" (
 echo Running %0 > "%INIT_TOOLS_LOG%"
 
 set /p DOTNET_VERSION=< "%~dp0DotnetCLIVersion.txt"
-if exist "%DOTNET_CMD%" goto :afterdotnetrestore
 
 :Arg_Loop
 if [%1] == [] goto :ArchSet
-if /i [%1] == [x86]         ( set ARCH=x86&&goto ArchSet)
+if /i [%1] == [x86]         ( set ARCH=x86)
+if /i [%1] == [-Debug-Intrinsics] ( set /p DOTNET_VERSION=< "%~dp0DotnetCLIVersion.netcoreapp.latest.txt")
```

Comment:
Can you do me a favor (doesn't need to be part of this PR)?

Can you update [https://github.com/dotnet/machinelearning/blob/master/docs/building/netcoreapp3.0-instructions.md](https://github.com/dotnet/machinelearning/blob/master/docs/building/netcoreapp3.0-instructions.md) for this new capability?  At the minimum step (2) in that doc is no longer needed.  Step (1) probably isn't necessary either anymore.

Reply:
sure I will do it today